src/main/resources/application.conf

```diff
@@ -4,8 +4,6 @@
 dbcpDataSource = {
   driverClassName = org.postgresql.Driver
   url = "jdbc:postgresql://google/avram?ssl=false&socketFactoryArg=broad-avram-dev:us-central1:avram&socketFactory=com.google.cloud.sql.postgres.SocketFactory"
-  username = "avram"
```

Comment:
Can this file go away completely if everything is in `app.conf` now?

Reply:
Probs